### DISCUSSION TITLE
correos masivos

### DISCUSSION THREAD

**Discussion by JGCODE47** [PHPMailer/PHPMailer#2586] [General]
hice un codigo con mailer para enviar unos email a muchos usuarios pero no funciona el $mail->addAddress lo meti dentro de un algoritmo que imprime esta variable muchas veces con los diferentes email pero jamás se envían estos

**Comment by Synchro** [comment]
I've not tried translating that but I think you want to send to a list, so please take a look at the mailing list example provided with PHPMailer.
